### PR TITLE
Replace context selector classes with Patternfly equivalents

### PR DESCRIPTION
**Pull request by srambach**
**What this PR does / why we need it**:

This PR replaces custom styling classes with the Patternfly equivalents for the context selector in the header. It also updates the styling to match current Patternfly.

**Which issue(s) this PR fixes** 

https://issues.redhat.com/browse/THREESCALE-6693

**Verification steps** 

Use the content selector in the masthead to verify it still operates. Inspect the elements to see that the context selector Patternfly classes are now included instead of custom "PopNavigation" classes.

**Special notes for your reviewer**:
image

### CODE REVIEW

File: app/javascript/src/patternflyStyles/pf4Base.css [hunk 1]
```diff
@@ -7,6 +7,8 @@
 
 @import '~@patternfly/patternfly/components/Page/page.css';
 @import '~@patternfly/patternfly/components/Nav/nav.css';
+/* @import '~@patternfly/patternfly/components/ContextSelector/context-selector.css'; */
```
Comment (damianpm) [inline app/javascript/src/patternflyStyles/pf4Base.css:10]: Do we need this commented line?

**Review by damianpm — CHANGES_REQUESTED** [review]
Can you please update the snapshots?
You can do it running:

`yarn jest -- -u `

File: app/javascript/src/patternflyStyles/pf4Base.css (hunk 1 above)
Comment (srambach) [reply app/javascript/src/patternflyStyles/pf4Base.css]: @damianpm I think that might actually be a better place for it so I put it back and removed the other include. Good catch. 
On the tests, jest still fails tests and doesn't create new snapshots. The purpose of this PR was an improvement for QE, so maybe tests need to be rewritten to use the PF classes? That's not my area, so I'm not sure.

Comment (damianpm) [reply app/javascript/src/patternflyStyles/pf4Base.css]: OK, I'll merge to a different branch, fix the tests, and then we can merge to master

**Review by damianpm — APPROVED** [review]
Merging to `context-selector-qe-classes`, then fixing tests

**Comment by damianpm** [comment]
@srambach check https://github.com/3scale/porta/pull/2373

**Comment by srambach** [comment]
@damianpm Thanks!

**Pull request merged**